examples/jose/update.py

```
@@ -0,0 +1,365 @@
+#!/usr/bin/env python3
+import os
+import sys
+import argparse
+import asyncio
+import logging
+import difflib
+from pathlib import Path
+
+from lightrag import LightRAG, QueryParam
```

Ruff (0.8.2)

`lightrag.QueryParam` imported but unused

Remove unused import: `lightrag.QueryParam`

(F401)